aptos-move/vm-genesis/src/lib.rs

```diff
@@ -384,6 +385,26 @@ fn exec_function(
         });
 }
 
+// Calculates the per-epoch rewards rate, represented as 2 separate ints (numerator and
+// denominator).
+#[cfg(feature = "aptos")]
+fn rewards_rate(genesis_config: &GenesisConfiguration) -> (u64, u64) {
+    const REWARDS_RATE_DENOMINATOR: u64 = 1_000_000_000;
+    // Multiplication before division to minimize rounding errors due to integer division.
+    // To convert the percentage to the fractional value, divide by 100, which can be folded
+    // into the constant denominator scaling factor.
+    let rewards_rate_numerator = genesis_config.rewards_apy_percentage
+        * (REWARDS_RATE_DENOMINATOR / 100)
```

Comment:
You don't need to divide by 100, just set the const value at 10_000_000.

Reply:
That would make the calculation harder to understand, and we need the same value (not divided by 100) for the denominator.

Reply:
Ah yes, I didn't see the denominator. I think it's better to multiply by 100 than dividing. For the calculus understanding in my opinion it doesn't change because we don't know why REWARDS_RATE_DENOMINATOR = 1_000_000_000. So set it to 10_000_000 doesn't change the understanding.